pkg/operation/hybridbotanist/addons.go

```diff
@@ -128,10 +138,11 @@ func (b *HybridBotanist) generateCoreAddonsChart() (*chartrenderer.RenderedChart
 		"cluster-autoscaler":  clusterAutoscaler,
 		"podsecuritypolicies": podsecuritypolicies,
 		"coredns":             coreDNS,
-		"kube-proxy":          kubeProxy,
-		"vpn-shoot":           vpnShoot,
-		"calico":              calico,
-		"metrics-server":      metricsServer,
+		"csi-" + b.ShootCloudBotanist.GetCloudProviderName(): csiPlugin,
```

Comment:
For better readability: please use `fmt.Sprintf()`

Reply:
done